Detectors/TOF/calibration/include/TOFCalibration/TOFChannelCalibrator.h

```diff
@@ -415,12 +211,54 @@ class TOFChannelCalibrator final : public o2::calibration::TimeSlotCalibration<T
   void setDoCalibWithCosmics(bool doCalibWithCosmics = true) { mCalibWithCosmics = doCalibWithCosmics; }
   bool doCalibWithCosmics() const { return mCalibWithCosmics; }
 
+  void setNThreads(int n) { mNThreads = std::min(n, NMAXTHREADS); }
+  int getNThreads() const { return mNThreads; }
+
+  void setStripFunction()
+  {
+    mStripOffsetFunction.clear();
+    for (int i = 0; i < 96; i++) {
+      int irow = i % 48;
+      int icolumn = i / 48;
+      if (i > 0) {
+        mStripOffsetFunction.append(Form("++ ("));
+      } else {
+        mStripOffsetFunction.append(Form("("));
+      }
+      bool kadd = false;
+      if (irow < 47) {
+        mStripOffsetFunction.append(Form("(x > %d && x < %d)", irow + icolumn * 48, irow + icolumn * 48 + 1));
+        kadd = true;
+      }
+      if (irow > 0) {
+        mStripOffsetFunction.append("-");
+        mStripOffsetFunction.append(Form("(x > %d && x < %d)", irow + icolumn * 48 - 1, irow + icolumn * 48));
+        kadd = true;
+      }
+      if (icolumn < 1) {
+        if (kadd) {
+          mStripOffsetFunction.append("+");
+        }
+        mStripOffsetFunction.append(Form("(x > %d && x < %d)", irow + 96, irow + 96 + 1));
+      } else {
+        mStripOffsetFunction.append("-");
+        mStripOffsetFunction.append(Form("(x > %d && x < %d)", irow + 96, irow + 96 + 1));
+      }
+      mStripOffsetFunction.append(Form(") "));
```

Comment:
This function allows to fit the differences between the peaks for each channel. Using such function allows to get better results since we have 96 channels per strip to be calibrated, but we can use 144 points (all pairs between adjacent channels, adjacent in X or Z).